### PR TITLE
Added login view

### PR DESCRIPTION
**Pull request by lukaszwasielewski**

### CODE REVIEW

**Comment by UlaR11** [comment]
Good job, but conflicts need to be resolved.

**Comment by UlaR11** [comment]
Looks ok.

File: src/main/resources/templates/login.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Website Font style -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```
Comment (RafalCiolek) [inline src/main/resources/templates/login.html]: Use head fragment

File: src/main/resources/templates/login.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                    <div class="cols-sm-10">
+                        <div class="input-group">
+                            <span class="input-group-addon"><i class="fa fa-lock fa-lg" aria-hidden="true"></i></span>
+                            <input type="password" class="form-control" name="password" id="password"
+                                   placeholder="Enter password"/>
+                        </div>
+                        <a href="forgot">Forgot your password?</a>
+                    </div>
+                </div>
+
```
Comment (RafalCiolek) [inline src/main/resources/templates/login.html]: useless space?

File: src/main/resources/templates/login.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+<div th:replace="fragments/header :: header "></div>
+
+<div class="container">
+    <div class="row main">
+        <div class="col-sm-3">
+
+        </div>
+        <div class="main-login main-center; col-sm-6">
+            <form class="form-horizontal" method="post" action="#">
+
```
Comment (RafalCiolek) [inline src/main/resources/templates/login.html]: one, max two lines is enough. :)

File: src/main/resources/templates/login.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                    <label for="email" class="cols-sm-2 control-label">Email</label>
+                    <div class="cols-sm-10">
+                        <div class="input-group">
+                            <span class="input-group-addon"><i class="fa fa-envelope fa" aria-hidden="true"></i></span>
+                            <input type="text" class="form-control" name="email" id="email"
+                                   placeholder="Enter email"/>
+                        </div>
+                    </div>
+                </div>
+
```
Comment (RafalCiolek) [inline src/main/resources/templates/login.html]: one, max two lines is enough. :)

**Review by LukaszSzetela — APPROVED** [review]

**Review by RafalCiolek — APPROVED** [review]

**Pull request merged**